src/Umbraco.Web.UI.Client/src/common/directives/components/property/umbproperty.directive.js

```diff
@@ -15,38 +15,31 @@ angular.module("umbraco.directives")
             restrict: 'E',
             replace: true,
             templateUrl: 'views/components/property/umb-property.html',
-            link: function (scope) {
+            link: function ($scope) {
 
-                scope.propertyEditorAPI = {};
+                $scope.propertyActions = [];
 
                 userService.getCurrentUser().then(function (u) {
                     var isAdmin = u.userGroups.indexOf('admin') !== -1;
-                    scope.propertyAlias = (Umbraco.Sys.ServerVariables.isDebuggingEnabled === true || isAdmin) ? scope.property.alias : null;
+                    $scope.propertyAlias = (Umbraco.Sys.ServerVariables.isDebuggingEnabled === true || isAdmin) ? $scope.property.alias : null;
                 });
             },
             //Define a controller for this directive to expose APIs to other directives
             controller: function ($scope, $timeout) {
 
                 var self = this;
+                self.propertyActions = [];
```

Comment:
This is unecessary and isn't actually doing anything. Below, in the `setPropertyActions` you are assigning the property actions directly to the $scope.